mozart++/system/file.hpp

```diff
@@ -0,0 +1,60 @@
+/**
+ * Mozart++ Template Library
+ * Licensed under MIT License
+ * Copyright (c) 2020 Covariant Institute
+ * Website: https://covariant.cn/
+ * Github:  https://github.com/covariant-institute/
+ */
+#pragma once
+
+#include <cstdio>
+#include <cstring>
+
+#ifdef _WIN32
+#include <Windows.h>
+#else
+#include <unistd.h>
+#endif
+
+namespace mpp {
+#ifdef _WIN32
```

Comment:
In Mozart++, you can use `MOZART_PLATFORM_WIN32` instead.

Reply:
ok